### DISCUSSION TITLE
How to display eww bar one monitor only

### DISCUSSION THREAD

**Discussion by FXCNN** [elkowar/eww#1051] [General]
Hi, I have of Monitors one by VGA and another HDMI, How do I tell it to show me eww bar on the HDMI monitor only.

**Comment by VehementHam** [comment]
Set the `monitor` property to the monitor ID. Here is an example:
```lisp
(defwindow example
           :monitor 0
           :geometry (geometry :x "0%"
                               :y "20px"
                               :width "90%"
                               :height "30px"
                               :anchor "top center")
           :stacking "fg"
           :reserve (struts :distance "40px" :side "top")
           :windowtype "dock"
           :wm-ignore false
  "example content")
```

**Comment by MyGitHubBlueberry** [comment]
Replace HDMI with exact monitor name.
```
:monitor '["HDMI"]'
```

**Reply by HeathRiewe** [reply]
Personally I have not been successful using connection names with eww on Wayland/Hyprland.  
directly using the model or ID works for me.
:monitor '["ASUS VS228"]'
:monitor '["U28E590"]'
:monitor 0
:monitor 1

a fair amount of info can be found with "hyprctl monitors"
